f3_test.go

```diff
@@ -108,26 +108,22 @@ func TestF3PauseResumeCatchup(t *testing.T) {
 
 	// Resuming node 1 should continue agreeing on instances.
 	env.resumeNode(1)
-	// make sure we're ahead of node 2, which may be ahead of node 0.
-	resumeInstance := env.nodes[2].currentGpbftInstance() + 1
+	// make sure we're ahead of node 1.
+	resumeInstance := env.nodes[1].currentGpbftInstance() + 1
 	env.waitForInstanceNumber(resumeInstance, 30*time.Second, false)
```

Comment:
We no longer have access to the cert-store while paused but... there wasn't much point in that anyways. This test is now a little less accurate (check then pause, instead of pause then check), but it should be good enough and shouldn't be flaky.